nodes/clusters/nodes-cluster-resource-levels.adoc

```diff
@@ -3,7 +3,6 @@
 = Estimating the number of pods your {product-title} nodes can hold
 include::modules/common-attributes.adoc[]
 
-
 toc::[]
 
```

Comment:
I'll just call it out once as it occurs often but it should not be of concern to us if there are multiple blank lines below `toc::[]`, correct?

Reply:
@bobfuru thank you for reviewing this! I noticed that too, but thought it best to focus for this PR on the blank lines prior to the `toc::[]` declaration in each assembly.

We could use a similar version of the script that I wrote to ensure that there is only one blank line after the `toc::[]` declaration too, in another set of PRs. My understanding is that this should not impact whether the TOCs are built. For now I think the important thing is to make sure that there is one blank line between the header and the TOC declaration in each assembly. This means that we can remove the unnecessary attributes from the `modules/common-attributes.adoc` file and still have the TOCs built successfully.

Reply:
I agree 💯  percent, @pneedle-rh. Thank you!